config/cscs.py

```diff
@@ -668,9 +668,9 @@
                             ]
                         },
                         {
-                            'name': 'mem-per-node',
+                            'name': 'memory',
                             'options': [
-                                '--mem={mem_per_node}'
```

Comment:
This you can still keep it as `mem_per_node`. The name of the resource can have multiple options, that's they should not necessarily be the same.